packages/genesys-spark-tokens/data/semantic/global.json

```diff
@@ -291,10 +291,15 @@
       "value": "{opacity.50}",
       "type": "opacity",
       "description": "The opacity we apply when a component is disabled."
+    },
+    "shroud": {
+      "value": "{opacity.80}",
+      "type": "opacity",
+      "description": "The default opacity for shroud elements like the one on modals."
     }
   },
   "interactive": {
-    "medium": {
+    "md": {
```

Comment:
Are these kinds of changes breaking changes? For example, `interactive.medium.size` is being used in the `forms_and_inputs.json` file.

Reply:
It shouldn't be a breaking change, all the references to `interactive.medium.size` in other JSONS should be displaying to `interactive.md.size`, those semantic tokens were introduced with themes so it shouldn't be used outside of references currently

https://github.com/MyPureCloud/genesys-spark/blob/4f8b5e5c707b63f44b29102ad9fcb4ce79a875a0/packages/genesys-spark-tokens/data/ui/forms_and_inputs.json#L1369